registry/handlers/referrers.go

```diff
@@ -195,6 +227,20 @@ func readlink(ctx context.Context, path string, stDriver driver.StorageDriver) (
 	return digest.Parse(string(content))
 }
 
+func generateLinkHeader(repoName, subjectDigest, artifactType string, lastDigests []string, nPage int, p int) string {
```

Comment:
`lastDigests` is never used.

Reply:
Forgot to delete this from legacy code. Deleted.